cmd/genkey.go

```diff
@@ -46,6 +47,7 @@ func NewGenkeyCmd() *cobra.Command {
 				cmd := exec.Command(
 					"openssl",
 					"req", "-nodes", "-x509", "-subj", fmt.Sprintf("/CN=%s/", name),
+					"-days", viper.GetString("expiration-in-days"),
```

Comment:
should we include a default?

Reply:
the cobra/viper setup sets the default to 365: https://github.com/kairos-io/enki/pull/58/files#diff-62ec5b9a8dabe763fd852506fa009728787133cc39e6c7618366532abe23a7eaR127